doc/man1/openssl-ca.pod.in

```diff
@@ -763,6 +762,11 @@ B<-enddate> and B<-days>) will be encoded as UTCTime if the dates are
 earlier than year 2049 (included), and as GeneralizedTime if the dates
 are in year 2050 or later.
 
+OpenSSL 1.1.1 introduced a new random generator (CSPRNG) with an improved
+seeding mechanism. The new seeding mechanism makes it dispensable to
```

Comment:
I'd suggest: `s/dispensable/unnecessary/`.
Dispensable seems odd here.

Reply:
Thanks for the hint, I'll fix it. Such things happen when one just looks up a term in the dictionary (here: [verzichtbar](https://www.dict.cc/?s=verzichtbar)) and chooses from a list of possible translations.

Reply:
Done in a4d887541fe2d28c183237274160b9e419f8b7a9.

Reply:
I suspected it was something like that.  English has a large number of similar meaning words each with their own undertones and nuances.